src/common/services/casApi.service.js

```diff
@@ -0,0 +1,69 @@
+import angular from 'angular';
+import 'angular-environment';
+import {Observable} from 'rxjs/Observable';
+import 'rxjs/add/observable/from';
+import 'rxjs/add/operator/map';
+import 'rxjs/add/operator/do';
+
+let serviceName = 'casApiService';
+
+class CasApi {
+
+  /*@ngInject*/
+  constructor( $http, envService ) {
+    this.$http = $http;
+    this.envService = envService;
+  }
+
+  http( config ) {
+    config.params = config.params || {};
+    if ( config.followLocation ) {
+      config.params.followLocation = true;
```

Comment:
I meant this is a cortex api feature https://touchpoint-developers.elasticpath.com/1.16.0/API-Reference/Follow-Location (might need to sign in to view).
> The FollowLocation query parameter reduces the number of requests required when performing POST operations. FollowLocation instructs Cortex to retrieve the resource referenced in the Location header and return it in the response body which eliminates the need to make another call to retrieve the resource. FollowLocation can also be used with zoom to further reduce the number of HTTP requests required.

I don't think it's a generic http thing... I couldn't find it quickly on Google anyways.